### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by appleconda**
I have updated the README.md. 
Changes: 
- Added browsing parameters table
- Added screenshot of the parameters. 
- Added supported operating system's logo (let me know if this should be kept or removed

- Changed the browsermon.conf to make it more readable.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -91,7 +91,7 @@ jobs:
                 name: browsermon_win_x64
                 path: artifact/
     
-    build_linux_x86:
+    build_linux_x64:
```
Comment (jdeunomatix) [inline .github/workflows/release.yml:94]: until now we were building 32-bit binaries ?

File: README.md [hunk 2]
```diff
@@ -14,6 +14,54 @@ Welcome to BrowserMon, the revolutionary browser monitoring tool designed to pro
...
+| `referrer`        | Referrer URL (if any).                             |
+| `url`             | URL of the webpage visited.                        |
+| `title`           | Title of the webpage visited.                      |
+| `visit_time`      | Time of the visit.                                 |
+| `visit_count`     | Number of times the URL was visited.               |
+
+BrowserMon in action!
+```json
+{
+  "hostname": "ABDULLAHCHA49D1",
```
Comment (jdeunomatix) [inline README.md:43]:
```suggestion
  "hostname": "HOST-A49D1",
```

File: README.md [hunk 3]
```diff
@@ -14,6 +14,54 @@ Welcome to BrowserMon, the revolutionary browser monitoring tool designed to pro
...
+  "os": "Windows",
+  "os_username": "appleconda",
```
Comment (jdeunomatix) [inline README.md:45]:
```suggestion
  "os_username": "jane_doe",
```

File: README.md [hunk 4]
```diff
@@ -14,6 +14,54 @@ Welcome to BrowserMon, the revolutionary browser monitoring tool designed to pro
...
+  "browser": "edge",
+  "browser_version": "119.0.2151.72",
+  "browser_db": "SQLite 3.35.5",
+  "profile_id": "",
+  "profile_title": "Profile 1",
+  "profile_username": "",
+  "profile_path": "C:\\Users\\appleconda\\AppData\\Local\\Microsoft\\Edge\\User Data\\Default",
```
Comment (jdeunomatix) [inline README.md:52]:
```suggestion
  "profile_path": "C:\\Users\\jane_doe\\AppData\\Local\\Microsoft\\Edge\\User Data\\Default",
```

File: README.md [hunk 5]
```diff
@@ -14,6 +14,54 @@ Welcome to BrowserMon, the revolutionary browser monitoring tool designed to pro
...
+  "username": "appleconda",
```
Comment (jdeunomatix) [inline README.md:53]:
```suggestion
  "username": "jane_doe",
```

**Review by jdeunomatix — CHANGES_REQUESTED** [review]
Minor edits

File: .github/workflows/release.yml (hunk 1 above)
Comment (appleconda) [reply .github/workflows/release.yml:94]: It was a typo, The job was making x64 binaries always.

**Review by jdeunomatix — APPROVED** [review]

**Pull request merged**